File: Beginner/numMirror/numMirror.c
#include <stdio.h>

int main(void){
	int n;

	while(1){
		scanf("%d",&n);

		if(n<0 || n>100000){
			printf("n must be 0 <= n <= 10^5\n");
		}else{
			printf("%d\n",n);
			break;
		}
	}

	return 0;
}

File: Beginner/numMirror/numMirror.java
import java.util.Scanner;

public class numMirror{
	public static void main(String[] args) {
		int n;
		Scanner scan = new Scanner(System.in);

		while(true){
			n = scan.nextInt();

			if(n<0 || n>100000){
				System.out.println("n must be 0 <= n <= 10^5");
			}else{
				System.out.println(n);
				break;
			}
		}
	}
}